orangecontrib/prototypes/widgets/owcontingency.py

```diff
@@ -55,19 +76,56 @@ def set_data(self, data):
                 self.rows = self.feature_model[0]
                 self.columns = self.feature_model[0]
                 self.openContext(data)
+                self.tableview.set_variables(self.rows, self.columns)
+                self.table = contingency_table(self.data, self.columns, self.rows)
+                self.tableview.update_table(self.table.X, formatstr="{:.0f}")
+        else:
+            self.tablemodel.clear()
```

Comment:
As per comment above,  ContingencyTable needs a method for clearing data in QStandardItemModel
``` 
self.tableview.clear()
```

Reply:
+1